### PR TITLE
FT: updating device maintenance logs related endpoints

### PR DESCRIPTION
**Pull request by danielogen**
**What does this PR do?**
This PR provide some improvement to the maintenance logs endpoint in device monitoring which will be consumed by the dashboard cards.  Also, updates some status codes.
**What JIRA task is related to this PR?**
This PR is related to [this](https://airqoteam.atlassian.net/browse/NET-78) JIRA card.
**How do I test out this PR?**
Git clone this repo `cd AirQo-api/src/device-monitoring`. Set up Python environment: `python3 -m venv env` (if not yet created), then activate it using this command: `source env/bin/activate`.
Then run `pip install -r requirements.txt` to get all the necessary package. Afterwards, run `flask run` to test out the endpoints using any REST client tool available to you.
**Docker**: if you in love with containers, here are the procedures: `docker build -t airqo-devicemonitor-api .`, followed by `docker run -d -p 8000:4001 airqo-devicemonitor-api`. Afterwards, point you, REST client, to `http://localhost:8000` and checkout the API documentation for details.
**Which endpoints should be/are ready for testing?**
base URL: http://localhost:4001
- [x] [GET] /api/v1/device/monitor/maintenance_logs
- [x] [GET] /api/v1/device/monitor/maintenance_logs/<device_name>
Take a look at the [API](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=1063367751) documentation for more details.

### CODE REVIEW

**Comment by danielogen** [comment]
> Looks good Daniel. I've made a couple of changes in your PR description i.e. changed the repo from /device to /device-monitoring and changed data in the second partial url to device. Hope that's okay

Thank you very much, Lillian. That's perfectly okay.

**Comment by lilly-muyama** [comment]
Wasn't this containerization done in the infrastructure meeting. Can this PR be updated with the Dockerfile @danielogen

**Comment by danielogen** [comment]
> Wasn't this containerization done in the infrastructure meeting. Can this PR be updated with the Dockerfile @danielogen

Hi @lilly-muyama, kindly review the updates. Thank you!

File: src/device-monitoring/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Inherit from the Python Docker image
```
Comment (Baalmart) [inline src/device-monitoring/Dockerfile:1]: Awesome addition.

**Pull request merged**